test/dropbox/run.sh

```diff
@@ -6,7 +6,10 @@ cd "$(dirname "$0")"
 . ../util.sh
 current_test=$(basename $(pwd))
 
-docker compose up -d
+sudo sed -i 's/SERVER_MODIFIED_1/'"$(date "+%Y-%m-%dT%H:%M:%SZ")/g" user_v2.yaml
```

Comment:
This sounds as if it's a job for `envsubst`, no? Also: I think it'd be nicer if the test didn't mutate a file that is checked into source control. Instead we could output a new, gitignored file so we get a stateless test?

Reply:
`envsubst` would need env vars to be set. The goal here not an external configuration or interaction with the environment, but to simply add the current date to the spec file, hence the `sed`.
Sure, we can do that.